### PR TITLE
[CF-516] Add in test spec for additions to getPartialFile site cascade/inheritance

### PR DESCRIPTION
**Pull request by rossbruniges**
I wanted to be sure that the new code that we've added to the getPartialFile in https://github.com/DigitalInnovation/connect-mustache-middleware/pull/4 is working as expected and not likely to cause issue to anyone not using it.

### CODE REVIEW

**Comment by nickvdyck** [comment]
Seems good to me, I'm only wondering if we don't need a test case for a failing call? And check the error?

File: index.js [hunk 1]
```diff
@@ -182,13 +182,12 @@ var MustacheEngine = {
...
      * @returns: string (site name ie. cfto)
      */
-
     getSiteName: function () {
         return argv.site ? argv.site : '';
     },
 
     /**
-     * getPartialFileContent
+     * getPartialFile
```
Comment (rossbruniges) [inline index.js]: Typo noticed by @nickvdyck - fixed on the sly as part of this PR :)

**Comment by rossbruniges** [comment]
That's a good point @nickvdyck - am wondering though. For this we're testing that try/catch works as expected.

Do we want to test core JavaScript or is that a test too far do you think?

**Comment by rossbruniges** [comment]
I decided that the try/catch was worth testing - and also added a check that the template inheritnace was working as expected.

There are now 4 tests - and I think that covers the important code paths!

**Pull request merged**